docs/content/getting-started.md

```diff
@@ -42,6 +42,7 @@ you should adjust to your own environment.
     !!! important
 
         In order to access applications in your guest clusters, the public zone must be routable.
+        Please skip this step if the public zone exists, otherwise it will affect the existing functions.
```

Comment:
Suggested revision for IBM Style: 

```
To access applications in your guest clusters, the public zone must be routable. If the public zone exists, skip this step. Otherwise, the public zone will affect the existing functions.
```